src/main/scala/com/ironcorelabs/document/DocumentEncryptResult.scala

```diff
@@ -26,19 +26,16 @@ case class DocumentEncryptResult(
 )(val underlyingBytes: Array[Byte])
 
 object DocumentEncryptResult {
-  def apply(der: com.ironcorelabs.sdk.DocumentEncryptResult): DocumentEncryptResult = {
-    val underlyingBytes = der.getEncryptedData
```

Comment:
Curious: what was the reason for getting rid of underlying bytes?

Reply:
The assignment wasn't really doing anything, was it? This isn't the `underlyingBytes` that's in the second parameter list of `DocumentEncryptResult`, this was just a temporary variable for `der.getEncryptedData`, so I just called that directly instead of assigning it to a variable.

Reply:
Yeah. That makes sense.